psydac/fem/splines.py

```diff
@@ -316,11 +312,32 @@ def eval_field_gradient( self, field, *eta , weights=None):
     # Other properties
     #--------------------------------------------------------------------------
     @property
+    def is_multipatch(self):
+        return False
+
+    @property
+    def is_vector_valued(self):
+        return False
+
+    # question [MCP 03.2025]: this is not in the FemSpace interface, redundant with is_vector_valued. keep it ?
+    @property
     def is_scalar( self ):
         """ Only scalar field is implemented for now.
         """
         return True
```

Comment:
How often is `is_scalar` used?

If we wanted to keep it for convenience, I would turn it into a concrete method of the `FemSpace` abstract base class which does simply
```python
@property
def is_scalar_valued(self):
    return not self.is_vector_valued
```

Reply:
no idea but good idea